server/hap/HapProcessCeilometer.cc

```diff
@@ -883,6 +896,24 @@ HatoholError HapProcessCeilometer::fetchItem(const MessagingContext &msgCtx,
 	return err;
 }
 
+HatoholError HapProcessCeilometer::fetchTrigger(const MessagingContext &msgCtx,
+						const SmartBuffer &cmdBuf)
+{
+	VariableItemTablePtr trigTablePtr;
+	HatoholError err =  getAlarmListReal(trigTablePtr);
```

Comment:
Suffix 'Real' may make readers some  unnecessary expectations.
For example, it must be a virtual or abstraction version of the function.